ruddock/modules/users/templates/memberlist.html

```diff
@@ -3,21 +3,48 @@
 {% block body %}
 <script src="{{ url_for('static', filename='js/tablesort.min.js') }}"></script>
 <h2>House Lists</h2>
-{# Show link to change current/alumni filtering #}
-{% if search_type == 'current' %}
-  Currently showing only current students. View
-  <a href="{{ url_for('users.show_memberlist') }}">all Rudds</a> or
-  <a href="{{ url_for('users.show_memberlist', search_type='alumni') }}">only alumni</a>.
-{% elif search_type == 'alumni' %}
-  Currently showing only alumni. View
-  <a href="{{ url_for('users.show_memberlist') }}">all Rudds</a> or
-  <a href="{{ url_for('users.show_memberlist', search_type='current') }}">only current students</a>.
-{% else %}
-  Currently showing all Rudds. View only
-  <a href="{{ url_for('users.show_memberlist', search_type='current') }}">current students</a> or
-  <a href="{{ url_for('users.show_memberlist', search_type='alumni') }}">alumni</a>.
-{% endif %}
-Click on the columns to sort.
+<script>
```

Comment:
idk javascript very well (i.e., at all). if anyone else is reading this, give it a once-over if you can